README.md

```diff
@@ -0,0 +1,163 @@
+ti-auth.js
+==========
+
+`ti-auth.js` is a small wrapper to integrate modules into the [Travel Intelligence Portal](https://travel-intelligence.com) providing:
+- **Authentication** - unauthorized users will be redirected to the TI Portal login page and will return after successful authentication.
+- **User Context** - user information, preferences and grants are made available to the application.
+- **Utility functions** - for signout and reauthorization.
+
+
+## Installation
+via Bower:
+
+    $ bower install git@github.com:travel-intelligence/ti-auth.js --save
+
+or via NPM:
+
+    $ npm install https://github.com/travel-intelligence/ti-auth.js --save-dev
+
+## Quick Start
+
+For ease of use the library exposes one global JavaScript object called **TiAuth**.
+
+After including the library in the in your application’s index.html file only 2 simple steps are needed to use `ti-auth.js` for authentication:
+
+1. **Configuration** - depending on the target environment (production/staging/local etc.) the Travel Intelligence API and the
```

Comment:
This sentence sounds confusing. The interface does not reflect anything about the environment so I’d rather not even mention it here. Also because we actually want to encourage cross-environment usage, *i.e.* use the staging API/Dashboard when working in  development.

How about simplifying this to:

---

1. **Configuration** - You need to specify the URLs of the Travel Intelligence API and the Portal you would like to use:

  ```javascript
  TiAuth.API_URL = 'https://api.travel-intelligence.com';
  TiAuth.DASHBOARD_URL = 'https://portal.travel-intelligence.com';
  ```

  *Note:* If you fail to provide these two URLs before calling `TiAuth.initialize()` an exception is thrown.